docs/environments/minikube.md

```diff
@@ -0,0 +1,35 @@
+# Minikube
+
+[Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) provides the quickest way to set-up clusters for
+use with the Federation v2 control-plane.
+
+**NOTE:** You will need to use a minikube version that supports deploying a
+kubernetes cluster >= 1.11. Currently there is no released version of minikube
+that supports kube v1.11 with profiles so you'll need to either:
+
+1. Build minikube from master by following these
+   [instructions](https://github.com/kubernetes/minikube/blob/master/docs/contributors/build_guide.md).
+2. Or use a recent CI build such as [this one from PR
+   2943](http://storage.googleapis.com/minikube-builds/2943/minikube-linux-amd64).
+
+Once you have minikube installed run:
+
+```bash
+minikube start -p cluster1 --kubernetes-version v1.11.0
+minikube start -p cluster2 --kubernetes-version v1.11.0
+```
+**NOTE:** Please make sure to set the correct context using the command below as this guide depends on it.
```

Comment:
This `**NOTE:**` probably makes more sense in the user guide as it applies to any environment setup. Feel free to address in a follow-up PR.